opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -224,6 +222,9 @@ def record(
     UPDATE_FUNCTION = record
 
 
+MetricT = TypeVar("MetricT", Counter, UpDownCounter, ValueRecorder)
```

Comment:
Should be `InstrumentT`?

Reply:
`Observer` types are "instruments" as well.